### PR TITLE
Added Dutch translation

### PR DESCRIPTION
**Pull request by Vistaus**
🙂

### CODE REVIEW

File: po/nl.po [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2023-07-11 01:41+0800\n"
+"PO-Revision-Date: 2023-08-14 20:44+0200\n"
+"Last-Translator: Heimen Stoffels <vistausss@fastmail.com>\n"
+"Language-Team: \n"
+"Language: nl\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"X-Generator: Poedit 3.3.2\n"
```
Comment (sabriunal) [inline po/nl.po:18]: Plural form information is missing. You can copy from Nautilus project.

https://gitlab.gnome.org/GNOME/nautilus/-/blob/main/po/ln.po#L21

Comment (sabriunal) [reply po/nl.po:18]: Plural form depends on the language.

**Comment by ghost** [comment]
Thanks. Merged.

**Comment by b00f** [comment]
@sabriunal

You can easily publish a new version:

1- Use `make pack` to create a zip file.
2- Publish it at https://extensions.gnome.org/#.

**Pull request merged**